inference-engine/src/cldnn_engine/ops/convolution.cpp

```diff
@@ -303,6 +303,78 @@ static void CreateDeformableConvolutionOp(Program& p, const std::shared_ptr<ngra
     }
 }
 
+static void CreateDeformableConvolutionOp(Program& p, const std::shared_ptr<ngraph::op::v8::DeformableConvolution>& op) {
```

Comment:
Can we extract generic code from old `CreateDeformableConvolutionOp(...)` and new functions into some common function `CreateDeformableConvolutionCommon`?

Reply:
done